modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -168,6 +169,12 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
         .void
     )
 
+  private val globalSnapshotProcessing: Stream[F, Unit] = Stream
+    .awakeEvery(10.seconds)
```

Comment:
I plan to extract all the intervals in `StateChannel` class to a config in the next PRs